### PR TITLE
Mark Gender Chart as Percentage

### PR DESCRIPTION
**Pull request by TaiWilkin**
## Overview

Adds "What is your gender" as question text that is recognized as a percentage chart type. 

Updates the data to the newest version. 

Connects #88 

### Demo

Screen Shot 2021-02-02 at 1 16 37 PM

## Testing Instructions

 * In the Netlify preview, select several locations.
 * Skim over the questions and confirm that the punctuation generally appears to be correct (for example, '* main*', with a space after the first asterisk, should not appear in the updated data). 
 * Select several questions, including 'What is your gender?', and navigate to the charts page.
 * Confirm that the percentage symbols appear with charts where it makes sense, and that 'What is your gender?' is treated as a percent.

### CODE REVIEW

File: src/dataproc/generate_data.py [hunk 1]
```diff
@@ -131,7 +132,7 @@ def generate_config(config_df, data_df, mode: str):
     ]
 
     # The value to use for type, based on the heuristic index above
-    values = ["stack", "pct", "ten", "hours", None]
+    values = ["stack", "pct", "pct", "ten", "hours", None]
```
Comment (fungjj92) [inline src/dataproc/generate_data.py:135]: Just to be sure, `"pct"` is supposed to be listed twice?

**Review by fungjj92 — APPROVED** [review]
App passed the testing instructions. Good to go pending one QC question below.

File: src/dataproc/generate_data.py (hunk 1 above)
Comment (TaiWilkin) [reply src/dataproc/generate_data.py:135]: Yep! It's because we're treating the gender question as a 'special exception percent'.

**Comment by TaiWilkin** [comment]
Thank you for all the reviews!

**Pull request merged**